src/controllers/category.js

```
@@ -15,3 +16,40 @@ const getCategories = async (req, res) => {
+
+  const subjectNames = await categoryService.getSubjectNamesByCategoryId(id, {
+    search,
+    page: parsedPage,
+    limit: parsedLimit,
+  });
+
+  res.status(200).json(subjectNames);
+};
+
+const getCategoryById = async (req, res) => {
+  const { id } = req.params;
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)